client/asset/dcr/dcr.go

```diff
@@ -4710,12 +4713,18 @@ func (dcr *ExchangeWallet) handleTipChange(ctx context.Context, newTipHash *chai
 		return
 	}
 
+	startHeight := prevTip.height + 1
```

Comment:
Why is this moved up to here?

Reply:
Just to keep `isMainchain` check as close as possible to `blockHeader` (where we basically got it from) - so it looks as single logical block of code. While for `startHeight` it doesn't really matter where it will be placed, all else equal typically prefer not to break those logical blocks.

<img width="1310" alt="image" src="https://user-images.githubusercontent.com/112318969/228412907-253e27dc-8be0-44bc-aec9-56e2a01fff1b.png">

Reply:
Seems pretty subjective. Try not to fix things that aren't broken. You can leave it now though.